### PR TITLE
created the admin _order-id view and update-admin-order controller

### PR DESCRIPTION
**Pull request by Sidahmeders**

### CODE REVIEW

File: infrastructure/web/views/admin/orders/_order-id.njk [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              لا
+            </label>
+          </div>
+        </div>
+
+        <div class="field">
+          <label class="label">مصاب بفيروس كورونا</label>
+          <div class="control">
+            <label class="radio">
+              <input type="radio" name="isCovid" value="true">
```
Comment (Abdelaziz18003) [inline infrastructure/web/views/admin/orders/_order-id.njk]: We need to check the right radio box (covid here and chronic deseases above) using `{{"checked" if order.isCovid}}` see /profile/orders/_order-id.njk for an example.

File: infrastructure/web/controllers/get-admin-orders.js [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
-      const { ownerId } = req.body
-      const ordersList = await listOrders({ ownerId })
-      res.render('admin/orders/index', { data: { orders: ordersList } })
+      const orderId = req.params.id
+      if (orderId) {
+        getAdminOrderById(req, res, listOrders, orderId)
+      } else {
+        // FIXME: THE USER_ID IS UNDEFINED FOR THE TIME BEIGN, SO WE ARE USING A HARD_CODED_ID
+        const user = req.user
+        const ownerId = user ? user.id : 'user_1'
```
Comment (Abdelaziz18003) [inline infrastructure/web/controllers/get-admin-orders.js]: we don't need to hard code ownerId any more, just login with an existing user ( I created some test users in fake DB) and the id will be present

File: infrastructure/web/controllers/get-admin-orders.js [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
+        const ordersList = await listOrders({ ownerId })
+        res.render('admin/orders/index', { data: { orders: ordersList } })
+      }
     } catch (err) {
       res.render('admin/orders/new', { errorMessages: [err.message] })
     }
   }
 }
+
+// NOTE: THIS METHOD CAN BE MOVED LATER TO A SEPRATE CONTROLLER
```
Comment (Abdelaziz18003) [inline infrastructure/web/controllers/get-admin-orders.js]: Yes it needs to be extracted out to a separate controller, I already did that for `get-user-product.js` controller you can take a look on it. Having that here, the getAdminOrders controller is doing too much work that it shouldn't be doing.

File: infrastructure/web/controllers/get-admin-orders.js [hunk 4]
```diff
@@ -1,11 +1,31 @@
...
+async function getAdminOrderById(req, res, listOrders, orderId) {
```
Comment (Abdelaziz18003) [inline infrastructure/web/controllers/get-admin-orders.js]: Yes it needs to be extracted out to a separate controller, I already did that for `get-user-product.js` controller you can take a look on it. Having that here, the getAdminOrders controller is doing too much work that it shouldn't be doing.
let's separate the concerns and have `getAdminOrders` for listing orders and `getAdminOrder` for getting a single order details.

**Review by Abdelaziz18003 — COMMENTED** [review]
@Sidahmeders thanks for the time and effort you put here. I left some feedback comments that you may want to have a look on :slightly_smiling_face:

**Comment by Sidahmeders** [comment]
sorry if I'm being late

**Comment by Abdelaziz18003** [comment]
@Sidahmeders no problem man, you are doing a great job :+1: :100:

**Pull request merged**